.env-mainnet-sample

```diff
@@ -67,3 +67,6 @@ EXTERNAL_LTC_ADDRESS=sample.ip.address
 #
 COMPOSE_PATH_SEPARATOR=:
 COMPOSE_FILE=docker-compose.yml:docker-compose.bitcoind.yml:docker-compose.litecoind.yml:docker-compose.override.yml
+
+# Path to rpc and identity certs
```

Comment:
The identity is not a cert, just a private and public key. Maybe we should rename the directory that they are in as well?

Reply:
to keys?